optuna/integration/pytorch_lightning.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+
+import optuna
+
+if optuna.type_checking.TYPE_CHECKING:
+    from typing import Dict  # NOQA
+
+try:
+    from pytorch_lightning.callbacks import EarlyStopping
+    _available = True
+except (ImportError, SyntaxError) as e:
```

Comment:
Could you tell me why `SyntaxError` is also caught here?

Reply:
PyTorch Lightning only supports Python 3.6 and higher. Importing PyTorch Lightning with Python 3.5 yields an `SyntaxError`. I agree it's far from obvious, let me leave a comment.

Reply:
Thank you.